sklearn/multioutput.py

```diff
@@ -598,3 +620,61 @@ def decision_function(self, X):
         Y_decision = Y_decision_chain[:, inv_order]
 
         return Y_decision
+
+
+class RegressorChain(_BaseChain, RegressorMixin, MetaEstimatorMixin):
+    """A multi-label model that arranges regressions into a chain.
+
+    Each model makes a prediction in the order specified by the chain using
+    all of the available features provided to the model plus the predictions
+    of models that are earlier in the chain.
+
```

Comment:
This should say "Read more in the User Guide"

Reply:
Followed the same procedure as already there in the codebase. 
`Read more in the :ref:`User Guide <regressorchain>`.` I am not sure whether this is what you intended as the hyper-link currently may not exist.

Reply:
You need to add `.. regressorchain:` before 
```
Regressor chain
===============
```
in `doc/modules/multiclass.rst`

Same for classifier chain